.azure-pipelines/noop-pipeline.yml

```diff
@@ -3,6 +3,7 @@ pr:
     exclude:
       - .
     include:
+      - tracer/dependabot/
```

Comment:
I'm not sure whether we need this one tbh, as we don't ever want to merge those PRs directly 🤔